### ISSUE TITLE
Menu selection colors are hard to see

### ISSUE THREAD

**Issue by jtsom** [maoma87/NightWolfTheme#10]
The latest change that adds better contrast when selecting something from a list (command pallet, file chooser, theme chooser, etc.) helps, but if just using the keyboard, it's all but impossible to see which is the current selection, or which is the selected command:

Night Wolf Gray: (gray is the current selection)
<img width="593" alt="image" src="https://user-images.githubusercontent.com/13488/59101293-823ed900-88f6-11e9-9ea8-5515eb09000d.png">

Night Wolf Blue: (blue is the current selection)
<img width="590" alt="image" src="https://user-images.githubusercontent.com/13488/59101258-650a0a80-88f6-11e9-9487-d502d73401a2.png">

the file/command chooser is a little better, but not much:
<img width="598" alt="image" src="https://user-images.githubusercontent.com/13488/59101326-a995a600-88f6-11e9-8a0b-1e871cede2ef.png">

**Comment by jakopo87** [comment]
I added this in settings.json as a workaround:
```
"workbench.colorCustomizations": {
    "[Night Wolf [dark gray]]": {
        "list.focusBackground": "#333"
    }
}
```

**Comment by jtsom** [comment]
For completeness I added:
```
        "[Night Wolf [dark blue]]": {
            "list.focusBackground": "#152739",
        },
```

**Comment by maoma87** [comment]
fixed in v1.2.2, thanks
